tomls/omnibus-base-mainnet-andromeda/perps/markets/snx.toml

```diff
@@ -21,7 +21,7 @@ snx_perps_endorsed_liquidator = "0x11233749514Ab8d00C0A5873DF7428b3db70030f"
 target = ["perpsFactory.PerpsMarketProxy"]
 fromCall.func = "owner"
 func = "createMarket"
-args = ["<%= settings.snx_perps_market_id %>", "Synthetix", "SNX"]
+args = ["<%= settings.snx_perps_market_id %>", "Synthetix Network Token", "SNX"]
```

Comment:
This will create new perps market. Better revert

Reply:
ohh yeah this is fixed in pr #3 sir

Reply:
Yeah I can see there are no extra txns so yes looks good.

Reply:
in the preview diff https://github.com/Synthetixio/Synthetix-Gitbook-v3/pull/464/files

![image](https://github.com/user-attachments/assets/4ffc7787-50e8-4392-a7fd-b3d9cb17b13a)

Reply:
yeah just to clarify, the snx and the sol markets are different in naming with respect to sepolia/mainnet... But the rest are aligned... So i had to make these changes here, when those markets are updated to the new configuration, in order to not create new markets